Modules/Sources/DesignSystem/Foundation/Color+DesignSystem.swift

```diff
@@ -35,8 +35,6 @@ public extension Color {
             private static let wordPress = colorWithModuleBundle(colorName: DesignSystemColorNames.Background.wordPress)
         }
 
-        public static let divider = colorWithModuleBundle(colorName: DesignSystemColorNames.divider)
```

Comment:
nit: it would be great to keep this property for convenience, similar to `UIColor.separator`.

Reply:
Hmm. Do you mean to keep this but use the background tertiary as its value? I am not sure about it. Reason being is that on Figma design will mention `Background.tertiary` and most likely the developer will use that value. If we use another naming even for convenience, that could break the sync between platforms. After all the whole point is that the names on Android, iOS and Figma to fully match. Wdyt?